<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createprogactivityscreen</class>
 <widget class="QWidget" name="createprogactivityscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="createProgText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Create Programming Activity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="createProgActivityButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>540</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Activity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>641</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Create Rubric :</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Section Description :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item row="1" column="0" rowspan="3">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Weight :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2" rowspan="2">
      <widget class="QPushButton" name="RemoveBtn">
       <property name="text">
        <string>Remove Section</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QTableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="AddBtn">
       <property name="text">
        <string>Add Section</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Tests Case:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="removeTestButton">
       <property name="text">
        <string>Remove Test</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="editTestButton">
       <property name="text">
        <string>Edit Test</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QListWidget" name="testList"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="addTestButton">
       <property name="text">
        <string>Add Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>571</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Activity Info:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5" rowspan="2">
      <widget class="QListWidget" name="typeList"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="programmingActivityLabel">
       <property name="text">
        <string>Programming Activity</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="gradeDirEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_3">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="activityTypeLabel">
       <property name="text">
        <string>Activity Type:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4" rowspan="2">
      <widget class="QListWidget" name="fileList"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Path to Grade File:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QPushButton" name="addFileButton">
       <property name="text">
        <string>Add File</string>
       </property>
      </widget>
     </item>
     <item row="10" column="6">
      <widget class="QPushButton" name="removeFileButton">
       <property name="text">
        <string>Remove File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Submission Files:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="activityLanguageLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Activity Language:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="4" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="pathToSubmissionsEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="languageBox">
       <item>
        <property name="text">
         <string>Select Language...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C++</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Java</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Python</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pathToSolEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="activityNameLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Activity Name:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Path to Solution:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="activityNameEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Path to submissions:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" rowspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Files:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>backButton</zorder>
  <zorder>createProgText</zorder>
  <zorder>createProgActivityButton</zorder>
  <zorder>pushButton</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <tabstops>
  <tabstop>activityNameEdit</tabstop>
  <tabstop>languageBox</tabstop>
  <tabstop>pathToSolEdit</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>pathToSubmissionsEdit</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>fileList</tabstop>
  <tabstop>addFileButton</tabstop>
  <tabstop>createProgActivityButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
